test/e2e/cases/basic.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/golang/glog"
+
+	"github.com/GoogleCloudPlatform/k8s-multicluster-ingress/app/kubemci/pkg/kubeutils"
+)
+
+// Creates a basic multicluster ingress and verifies that it works as expected.
+// TODO(nikhiljindal): Use ginkgo and gomega and possibly reuse k/k e2e framework.
+func RunBasicCreateTest() {
+	// Validate inputs and instantiate required objects first to catch errors early.
+	// TODO(nikhiljindal): User should be able to pass kubeconfig.
+	kubeconfig := "kubeconfig"
```

Comment:
change the variable name to kubeConfigPath? (to clarify it's a file and not the contents)

Reply:
Done